maxim/logger/livekit/store.py

```
@@ -0,0 +1,220 @@
+        with self._lock:
+            for entry in self.mx_live_kit_session_store:
+                if "rt_session_id" in entry and entry["rt_session_id"] == rt_session_id:
+                    return entry
+            return None
+
+    def set_session(self, entry: SessionStoreEntry):
+        with self._lock:
+            if "agent_session_id" in entry:
+                # find the entry and replace
+                for i, e in enumerate(self.mx_live_kit_session_store):
+                    if e["agent_session_id"] == entry["agent_session_id"]:                        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)